sdk/python/setup.py

```diff
@@ -32,6 +32,6 @@ def readme():
       install_requires=[
           'pulumi>=0.17.1,<0.18.0',
           'requests>=2.21.0,<2.22.0',
-          'pyyaml>=3.13,<3.14',
+          'pyyaml>=4.2b1,<4.3',
```

Comment:
What's the `b` mean?

Reply:
Not sure, but that's what the security alert recommended: https://github.com/pulumi/pulumi-kubernetes/network/alert/sdk/python/setup.py/pyyaml/open